internal/plugins/openshift/v1/plugin.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"sigs.k8s.io/kubebuilder/v2/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/v2/pkg/plugin"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins"
+)
+
+const (
+	pluginName = "openshift" + plugins.DefaultNameQualifier
```

Comment:
This becomes `openshift.sdk.operatorframework.io`, which isn't really correct. Perhaps something like `openshift.sdk.openshift.io`, which is repetitive but mirrors the structure of the upstream fq name?

Reply:
I'm actually okay with `openshift.sdk.operatorframework.io`

Reply:
Thoughts on `sdk.x-openshift.io`? I don't think it's ok to use `operatorframework.io` here since this is technically not an OF plugin.